target/linux/bcm63xx/image/bcm63xx.mk

```diff
@@ -3,14 +3,19 @@
 # BCM33XX/BCM63XX Profiles
 #
 
+DEVICE_VARS += HCS_MAGIC_BYTES HCS_REV_MIN HCS_REV_MAJ
+DEVICE_VARS += BLOCK_SIZE FLASH_MB IMAGE_OFFSET
```

Comment:
I wonder whether we could use BLOCKSIZE from image.mk instead ...

Reply:
Hmm maybe, but I don't know enough about CFE to to understand the image recipe. BLOCK_SIZE may have a different meaning than BLOCKSIZE.